eng/common/pipelines/templates/steps/check-spelling.yml

```diff
@@ -0,0 +1,38 @@
+# Checks spelling of files that changed between the current state of the repo
+# and some ref (branch, tag, etc.) or commit hash. Only runs on PRs.
+# ContinueOnError - true: Pipeline warns on spelling error 
+#                   false: Pipeline fails on spelling error
+# TargetRef       - Target ref (e.g. master) to compare to create file change
+#                   list
+# CspellConfigPath - Path to cspell.json config location
+
+parameters:
+  ContinueOnError: true
+  TargetRef: $(System.PullRequest.TargetBranch)
```

Comment:
Do we expect someone to change this as part of the template yml? Perhaps we should just use the variable and guard against running this step outside of a pull request.

Reply:
The default is sufficient. This is added for configurability to prevent someone from having to touch eng/common later.